cypress/support/commands.js

```diff
@@ -48,6 +46,17 @@ Cypress.Commands.add('rendererApi', (...args) => {
 })
 
 Cypress.Commands.add('setZclProperties', () => {
-  cy.get('[data-test="gen-template"]').click()
-  cy.get('[data-test="login-submit"]').click()
-})
\ No newline at end of file
+  cy.get('body').then(($body) => {
+    // The above doesn't work. It does find the buttons, even
+    // if they are hidden, obviously....
+    /*if ($body.find('[data-test="gen-template"]').length) {
```

Comment:
Do we need to update the tests to account for this behavior. With single package and without single package?

Reply:
We need to do something. tried the above and failed. Not sure what else to try. I'll have to study up cypress a bit more.